Automattic/jetpack
# Paypal Block: Fix issues with site editor

Fixes https://github.com/Automattic/jetpack/issues/19486

#### Changes proposed in this Pull Request:

* Fix an issue that stops the paypal block from rendering in the front end when used within the site editor.
* Fix an issue where the product image would be huge in the editor
* Fix form rendering issues so the form renders consistently in the post and site editor

Before in site editor:

![114128156-1472f680-993f-11eb-9d65-27b34cdfff14](https://user-images.githubusercontent.com/1464705/115470566-604d5780-a1eb-11eb-978a-7bffa5c1431f.png)

After in site editor:

<img width="633" alt="Screen Shot 2021-04-20 at 3 09 47 PM" src="https://user-images.githubusercontent.com/1464705/115470588-6b07ec80-a1eb-11eb-8c60-b3d18e4d74ec.png">


#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Activate TT1-blocks
* Open the site editor and insert a paypal block.
* Add a product image and confirm the size looks correct.
* Fill out the product details and preview the template on the front-end. Confirm the product is visible.
* Confirm the form renders correctly in the editor as per the screen grab above.
* Do the same in the post editor and confirm everything works correctly.
